apiserver/apiserver.go

```diff
@@ -783,6 +792,11 @@ func (srv *Server) endpoints() []apihttp.Endpoint {
 		tracked:         true,
 		unauthenticated: true,
 		noModelUUID:     true,
+	}, {
+		pattern:         "/cli",
```

Comment:
I'm not overly enthused with the idea of calling the endpoint "/cli". Perhaps something like 'repl' ?

Reply:
Strictly speaking, each API request is separate from the one before and causes a new CLI command instance to be created and run. So it doesn't behave as a repl in that sense, ie it's not an (interactive) loop

Reply:
I though about "embedded" or "embeddedcli" but these seemed overly long.
The idea behind "cli" was to reflect the purpose (embedded cli for dashboard or any client really).

Reply:
Changed to /commands for now